napari_arboretum/plugin.py

```diff
@@ -42,113 +28,53 @@ class Arboretum(QWidget):
 
     def __init__(self, viewer: napari.Viewer, parent=None):
         super().__init__(parent=parent)
-
-        # store a reference to the viewer
-        self._viewer = viewer
+        self.viewer = viewer
+        self.plotter: TreePlotterQWidgetBase = PyQtGraphPlotter()
 
         # build the canvas to display the trees
         layout = QVBoxLayout()
-        plot_widget = pg.GraphicsLayoutWidget()
-        self.plot_view = plot_widget.addPlot(
-            title="Lineage tree", labels={"left": "Time"}
-        )
-        self.plot_view.hideAxis("bottom")
-        layout.addWidget(plot_widget)
+        layout.addWidget(self.plotter.get_qwidget())
         layout.setAlignment(Qt.AlignTop)
         layout.setSpacing(4)
         self.setMaximumWidth(GUI_MAXIMUM_WIDTH)
         self.setLayout(layout)
 
-        # hook up an event to find Tracks layers if the layer list changes
-        self._viewer.layers.events.changed.connect(self._get_tracks_layers)
-
-        # store the tracks layers
-        self._tracks_layers: List[napari.layers.Tracks] = []
-        self._get_tracks_layers()
+        # hook up an event to update the list of tracks layers if the layer
+        # list changes
+        self.viewer.layers.events.changed.connect(self.update_tracks_layers)
+        self.tracks_layers: List[napari.layers.Tracks] = []
+        self.update_tracks_layers()
 
-    def _get_tracks_layers(self, event=None):
+    def update_tracks_layers(self, event=None):
```

Comment:
Because it saves typing an underscore each time a developer wants to use the variable/method, and I don't think any of the underlying code is public facing anyway (since the primary function of the package is to be a GUI), so it removes the slight friction of having to worry about making stuff public/private.